habitat-lab/habitat/sims/habitat_simulator/sim_utilities.py

```diff
@@ -13,6 +13,18 @@
 from habitat.sims.habitat_simulator.debug_visualizer import DebugVisualizer
 
 
+def object_shortname_from_handle(object_handle: str) -> str:
```

Comment:
we have a similar function in hab-llm: https://github.com/facebookresearch/habitat-llm/blob/main/dataset_generation/benchmark_generation/generate_instructions.py#L62 Not sure if it is worth making this a utils function in one place..

Reply:
Agreed. Once this merges, I'll refactor downstream code to use this util.